airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/JdbcStateManager.java

```diff
@@ -54,12 +54,16 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(JdbcStateManager.class);
 
   private final Map<String, CursorInfo> streamNameToCursorInfo;
+  private Boolean isCdc;
+  private final JdbcCdcStateManager cdcStateManager;
 
   public static JdbcState emptyState() {
     return new JdbcState();
   }
 
   public JdbcStateManager(JdbcState serialized, ConfiguredAirbyteCatalog catalog) {
+    this.cdcStateManager = new JdbcCdcStateManager(serialized.getCdcState());
+    this.isCdc = serialized.getCdc();
```

Comment:
instead of leaving the possibility for it to be null, why don't you clean it up and make it false or true if it null (I assume false)?